Assets/Scripts/Entities/Entity.cs

```diff
@@ -22,7 +22,10 @@ public Vector2 position
     public abstract void Update();
 
     // This is where input is received
-    public abstract void HandleInstruction(Instruction instruction);
+    public virtual void HandleInstruction(Instruction instruction)
+    {
+        this.transform.position = GameManager.GetTransformForInstruction(instruction, transform.position);
```

Comment:
In the current implementation, this will teleport the Entity to the target position. In the future, we'll use Entity.UpdateMovement() to do the smooth movement animation. Currently, UpdateMovement() lives inside PlayerController but we'll want to move it to Entity so that both Player and Enemy can use it.